tests/sanity/ignore-2.10.txt

```diff
@@ -82,3 +83,4 @@ plugins/modules/zos_tso_command.py import-2.6!skip # Python 2.6 is unsupported
 plugins/modules/zos_gather_facts.py validate-modules:missing-gplv3-license # Licensed under Apache 2.0
 plugins/modules/zos_gather_facts.py compile-2.6!skip # Python 2.6 is unsupported
 plugins/modules/zos_gather_facts.py import-2.6!skip # Python 2.6 is unsupported
+plugins/modules/zos_job_submit.py pylint:catching-non-exception
```

Comment:
Is this redundant entry necessary?

Reply:
Good catch, i was appending to the end of the file because i wanted an easy place to keep tweaking it and forgot to remove it. On that note, I had to add the exemption to all files not just 2.11 onward because it would fail, I think there is a bug in the tooling that reads any ignore txt when there are multiple, ill be opening an issue later with the community for this bug.